exporter/otlp-metrics/opentelemetry-exporter-otlp-metrics.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'googleapis-common-protos-types', '~> 1.3'
-  spec.add_dependency 'google-protobuf', '~> 3.14'
+  spec.add_dependency 'google-protobuf', '>= 3.14', '< 5.0'
```

Should this also be:
```suggestion
  spec.add_dependency 'google-protobuf', '>= 3.18', '< 5.0'
```
or:
```suggestion
  spec.add_dependency 'google-protobuf', '>= 3.18'
```